.github/ISSUE_TEMPLATE/suggest_layer2.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Suggest a Layer 2
+about: Suggest a Layer 2 to list on ethereum.org
+title: ""
+labels: "feature :sparkles:, content :fountain_pen:"
+assignees: ""
+---
+
+Before suggesting a layer 2, make sure you've read [our listing policy](/contributing/adding-layer-2/).
+
+Only continue with the issue if your layer 2 meets the criteria listed there.
+
+If it does, complete the following information which we need to accurately list the layer 2.
+
+**Name**
+
+<!-- Please provide the official name of the layer 2-->
+
+**Logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Description**
+
+<!-- Please provide a 1-2 sentence of the layer 2-->
+
+**Website**
+
+<!-- Please provide a URL to the website for the layer 2 -->
+
+**Developer documentation**
+
+<!-- Please provide a URL to developer docs for the layer 2 -->
+
+**L2Beat page**
+
+<!-- Please provide the link to the layer 2s page on L2BEAT-->
+
+**Open source**
+
+<!-- If yes, please provide a link to the repository -->
+
+**Type of layer 2 (Optimistic, ZK, State channel)**
+
+<!-- Please provide some information on what kind of layer 2 this is -->
+
+**Purpose / use case**
+
+<!-- Please provide some information on the use case for this layer 2 -->
+
+**Bridges**
+
+<!-- Please provide some information on how to bridge assets onto this layer 2 -->
+
+**Exchange support**
+
+<!-- If there is exchange support for deposits/withdrawals, please provide information for this -->
+
+**Link to dapp ecosystem (if applicable)**
+
+<!-- Please provide a link to the dapp ecosystem -->
+
+**Token contract list (if applicable)**
+
+<!-- Since assets will have a new address on layer 2, if there is a token list resource available please share  -->
+
+**How long has this project been live?**
+
+<!-- Please provide an approximate date for when the layer 2 went live -->
+
+**Is this project a fork?**
+
+<!-- Please provide information about if this project is a fork -->
+
+**What wallets support this layer 2?**
+
+<!-- Please list wallets that support this network -->
+
+**Is this project externally audited?**
+
+<!-- Please provide a link to a report or repo. If you haven't been audited but think your layer 2 should be listed anyway, explain here. -->
+
+**TVL history of the layer 2**
```

Comment:
Similarly here, I might open this up for people to include links to broader usage metrics. See my suggestion below.